emcc.py

```diff
@@ -1246,6 +1199,20 @@ def get_last_setting_change(setting):
     # Check if a target is specified on the command line
     specified_target, args = find_output_arg(args)
 
+    if os.environ.get('EMMAKEN_JUST_CONFIGURE') or 'conftest.c' in args:
+      # configure tests want a more shell-like style, where we emit return codes on exit()
+      shared.Settings.EXIT_RUNTIME = 1
+      # use node.js raw filesystem access, to behave just like a native executable
+      shared.Settings.NODERAWFS = 1
+      # Add `#!` line to output JS and make it executable.
+      shared.Settings.EXECUTABLE = 1
+      # Autoconf expects the executable output file to be called `a.out`
+      default_target_name = 'a.out'
```

Comment:
another option, which I'm not sure would be better, is to not do this here, and not make the other changes for `.out` suffix support, and instead if `EXECUTABLE` is set to do something at the very end of compilation (rename or copy `a.out.js` to `a.out`).

Reply:
I think I prefer this way rather than doing more copying/renaming.

I might even one day see if we can change the default globally to match clang/gcc default output name .. but that is much bigger change :)